modules/serverless-ossm-setup.adoc

```diff
@@ -177,6 +178,167 @@ spec:
 $ oc apply -f <filename>
 ----
 
+. Install Knative Eventing by creating the following `KnativeEventing` custom resource definition (CRD), which also enables the Istio integration:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing 
+spec:
+  config:
+    features:
+      istio: enabled <1>
+  workloads:
+  - name: pingsource-mt-adapter
+    annotations:
+      "sidecar.istio.io/inject": "true" <2>
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: imc-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: mt-broker-ingress
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: mt-broker-filter
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+----
+<1> Enables Eventing istio controller to create a `DestinationRule` for each InMemoryChannel or KafkaChannel service.
+<2> Enables sidecar injection for Knative Eventing pods.
+
+:FeatureName: The Knative Eventing integration with {SMProductShortName}
+include::snippets/technology-preview.adoc[leveloffset=+4]
+
+. Apply the `KnativeEventing` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Install Knative Kafka by creating the following `KnativeKafka` custom resource definition (CRD), which also enables the Istio integration:
++
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: knative-eventing
+spec:
+  channel:
+    enabled: true
+    bootstrapServers: <bootstrap_servers> <1>
+  source:
+    enabled: true
+  broker:
+    enabled: true
+    defaultConfig:
+      bootstrapServers: <bootstrap_servers> <1>
+      numPartitions: <num_partitions>
+      replicationFactor: <replication_factor>
+    sink:
```

Comment:
indentation, `sink: enabled:true` should be on the same level as `source:`, not inside `broker:`

Reply:
Incorporated! Thank you!